### PR TITLE
docs: added documentation typedoc comments for input-group and combobox-input components

### PR DESCRIPTION
**Pull request by InnaAtanasova**
#### Please provide a link to the associated issue.

part of [#687](https://github.com/SAP/fundamental-ngx/issues/687)

#### Please provide a brief summary of this pull request.

added documentation typedoc comments for  input-group and combobox-input components

### CODE REVIEW

File: library/src/lib/combobox-input/combobox-input.component.ts [hunk 1]
```diff
@@ -14,12 +14,15 @@ import { NG_VALUE_ACCESSOR } from '@angular/forms';
 })
 export class ComboboxInputComponent extends SearchInputComponent {
```
Comment (MattL75) [inline library/src/lib/combobox-input/combobox-input.component.ts]: document class

File: library/src/lib/input-group/input-group-number.component.ts [hunk 2]
```diff
@@ -16,44 +16,58 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
     ]
 })
 export class InputGroupNumberComponent implements ControlValueAccessor {
```
Comment (MattL75) [inline library/src/lib/input-group/input-group-number.component.ts]: document class

File: library/src/lib/input-group/input-group-search.component.ts [hunk 3]
```diff
@@ -16,34 +16,46 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
     ]
 })
 export class InputGroupSearchComponent implements ControlValueAccessor {
```
Comment (MattL75) [inline library/src/lib/input-group/input-group-search.component.ts]: document class

File: library/src/lib/input-group/input-group.component.ts [hunk 4]
```diff
@@ -13,59 +13,82 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
     ]
 })
 export class InputGroupComponent implements ControlValueAccessor {
```
Comment (MattL75) [inline library/src/lib/input-group/input-group.component.ts]: document class

File: library/src/lib/combobox-input/combobox-input.component.ts (hunk 1 above)
Comment (InnaAtanasova) [reply library/src/lib/combobox-input/combobox-input.component.ts]: done

File: library/src/lib/input-group/input-group-number.component.ts (hunk 2 above)
Comment (InnaAtanasova) [reply library/src/lib/input-group/input-group-number.component.ts]: done

File: library/src/lib/input-group/input-group-search.component.ts (hunk 3 above)
Comment (InnaAtanasova) [reply library/src/lib/input-group/input-group-search.component.ts]: done

File: library/src/lib/input-group/input-group.component.ts (hunk 4 above)
Comment (InnaAtanasova) [reply library/src/lib/input-group/input-group.component.ts]: done

**Pull request merged**